1-org/modules/cai-monitoring/main.tf

```diff
@@ -162,9 +162,8 @@ module "cloud_function" {
   service_config = {
     service_account_email = google_service_account.cloudfunction.email
     runtime_env_variables = {
-      ROLES            = join(",", var.roles_to_monitor)
-      SOURCE_ID        = google_scc_source.cai_monitoring.id
-      LOG_EXECUTION_ID = "true"
```

Comment:
I saw the discussion about removing `LOG_EXECUTION_ID` to potentially resolve #1306, but it appears the provider upgrade solved the issue. Was removing this still intentional or an oversight?